packages/@aws-cdk/aws-codecommit/lib/repository.ts

```diff
@@ -343,14 +368,9 @@ export class Repository extends RepositoryBase {
       service: 'codecommit',
       resource: this.physicalName,
     });
-  }
-
-  public get repositoryCloneUrlHttp() {
-    return this.repository.attrCloneUrlHttp;
-  }
-
-  public get repositoryCloneUrlSsh() {
-    return this.repository.attrCloneUrlSsh;
+    this.repositoryCloneUrlHttp = this.repository.attrCloneUrlHttp;
+    this.repositoryCloneUrlSsh = this.repository.attrCloneUrlSsh;
+    this.repositoryCloneUrlGrc = Repository.makeCloneUrl(Stack.of(this), this.repositoryName, 'grc');
```

Comment:
Now that the getters have been substituted with fields (thanks!), can we make `repository` a local variable in the constructor, instead of a field?